lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_dynamo_SUITE.erl

```diff
@@ -186,15 +187,22 @@ parse_and_check(ConfigString, BridgeType, Name) ->
     Config.
 
 create_bridge(Config) ->
-    BridgeType = ?config(dynamo_bridge_type, Config),
-    Name = ?config(dynamo_name, Config),
-    TDConfig = ?config(dynamo_config, Config),
-    emqx_bridge:create(BridgeType, Name, TDConfig).
+    create_bridge(Config, _Overrides = #{}).
 
-delete_bridge(Config) ->
+create_bridge(Config, Overrides) ->
```

Comment:
Nit. Is there a code that uses `create_bridge/2`?

Reply:
During my attempts, I tried to customize a few settings, ended up not using this one yet :see_no_evil: